### ISSUE TITLE
Cannot Import Button from CDN

### ISSUE THREAD

**Issue by nanarino** [patternfly/patternfly-elements#2245]
### Impacted component(s)

- 

### Steps to reproduce

```html
<script type="module" src="https://unpkg.com/@patternfly/pfe-button@next?module"></script>

<pfe-button>
  <button>My Button</button>
</pfe-button>
```
See error http 500 when GET `pfe-icon.js?module`    
but GET `pfe-icon.js` was normal

### Screenshots
![image](https://user-images.githubusercontent.com/49119482/204333081-087ae271-714e-4a21-bfd5-c57b73921988.png)

**Comment by bennypowers** [comment]
Thanks for the issue. You can expect this to be definitively resolved by #2134 

until then, it's possible this is an issue with unpkg support for export maps. Have you tried using the [jspm](https://jspm.dev) cdn with import maps?

**Comment by nanarino** [comment]
ohh，incredibly useful，多谢。
